packages/bot-skeleton/src/services/tradeEngine/trade/OpenContract.js

```diff
@@ -51,17 +52,17 @@ export default Engine =>
 
         subscribeToOpenContract(contract_id = this.contractId) {
             this.contractId = contract_id;
-            doUntilDone(() => this.api.send({ proposal_open_contract: 1, contract_id, subscribe: 1 }))
+            doUntilDone(() => api_base.api.send({ proposal_open_contract: 1, contract_id, subscribe: 1 }))
                 .then(data => {
                     const { populateConfig } = DBotStore.instance;
                     populateConfig(data.proposal_open_contract);
                     this.openContractId = data.proposal_open_contract.id;
                 })
                 .catch(error => {
                     if (error.error.code !== 'AlreadySubscribed') {
-                        doUntilDone(() => this.api.send({ proposal_open_contract: 1, contract_id, subscribe: 1 })).then(
-                            response => (this.openContractId = response.proposal_open_contract.id)
-                        );
+                        doUntilDone(() =>
+                            api_base.api.send({ proposal_open_contract: 1, contract_id, subscribe: 1 })
+                        ).then(response => (this.openContractId = response.proposal_open_contract.id));
```

Comment:
@sandeep-deriv maybe we create a reusable request object for both queries if the body exactly the same here?  Something like this? 
`const query_subscribe_open_contract = { 
  proposal_open_contract: 1, 
  contract_id, 
  subscribe: 1
}`